ComponentKit/StatefulViews/CKStatefulViewReusePool.mm

```diff
@@ -56,6 +90,8 @@ void addView(UIView *view)
 @implementation CKStatefulViewReusePool
 {
   std::unordered_map<std::pair<__unsafe_unretained Class, id>, FBStatefulReusePoolItem, PoolKeyHasher> _pool;
+  std::unordered_map<std::pair<__unsafe_unretained Class, id>, FBStatefulReusePoolItem, PoolKeyHasher> _pendingPool;
```

Comment:
Meh, this whole thing is a reuse pool. I don't think that improves readability at all. Up to you if you want to change it later though.